tyk-pro/templates/bootstrap-pre-delete.yaml

```diff
@@ -28,28 +28,19 @@ spec:
     spec:
       serviceAccountName: k8s-bootstrap-role
       containers:
-        - name: bootstrap-pre-delete
-          image: bash:5.1
+        - name: bootstrap-tyk
```

Comment:
WDYT updating it as `bootstrap-tyk-pre-delete`?

Reply:
Done.